components/ProposalCard.tsx

```diff
@@ -84,39 +102,57 @@ const ProposalCard = ({
           </span>
           {status ?? "Rejected"}
         </span>
-        <span
-          className="ml-8 truncate font-light text-zinc-300 md:ml-0"
-          style={{
-            minWidth: "7rem",
-          }}
-          title={content
-            .map(v => {
-              return "executeLambda" in v &&
+        <div className="ml-8 flex items-center space-x-4 md:ml-0">
+          {!!dapp && (
+            <div className="relative h-6 w-6 shrink-0">
+              {(() => {
+                switch (dapp) {
```

Comment:
Maybe make this switch in `identifyDapp.ts` or component for the config. The goal is centralized the necessary change together. so we don't need to go everywhere for the adding one dapp.